### ISSUE TITLE
[Feature Request] flow_from_directory like for TFRecord

### ISSUE THREAD

**Issue by chiboreache** [keras-team/keras#14083]
Would be nice to make it less tough.. 

AutoML CSV with labels, for example, so much easier for understanding 

`ImageDataGenerator.flow_from_directory()` workflow is just nobrain and awesome! meanwhile `TFRecord` and `tf.Example` quality of it official docs -- *makes me very unhappy*

**Comment by jvishnuvardhan** [comment]
@chiboreache Sorry for late response. Is this still an issue for you. 

There is a more detailed [tutorial](https://www.tensorflow.org/tutorials/load_data/tfrecord) on `TFRecord` and `tf.train.Example`.  Can you please check it and let us know if there is anything missing. 

Also, you can contribute to update the docs or code through PRs. Thanks!
